File: workspace/data-model/src/main/java/de/pfiva/data/model/survey/Question.java
package de.pfiva.data.model.survey;

import java.io.Serializable;
import java.util.List;

public class Question implements Serializable {

	private static final long serialVersionUID = 1L;
	
	private int id;
	private String question;
	private String questionType;
	private List<Option> options;
	
	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getQuestion() {
		return question;
	}

	public void setQuestion(String question) {
		this.question = question;
	}

	public String getQuestionType() {
		return questionType;
	}

	public void setQuestionType(String questionType) {
		this.questionType = questionType;
	}

	public List<Option> getOptions() {
		return options;
	}

	public void setOptions(List<Option> options) {
		this.options = options;
	}
	@Override
	public String toString() {
		return "Question [id=" + id + ", question=" + question + ", questionType=" + questionType + ", options="
				+ options + "]";
	}
}

File: angular_workspace/pfiva-web/src/app/data-model/survey/Question.ts
import { Option } from "./Option";

export class Question {

	private id: number;
	private question: string;
	private questionType: string;
	private options: Option[];

	public get Id(): number {
		return this.id;
	}
	public set Id(value: number) {
		this.id = value;
	}
	public get Question(): string {
		return this.question;
	}
	public set Question(value: string) {
		this.question = value;
	}
	public get QuestionType(): string {
		return this.questionType;
	}
	public set QuestionType(value: string) {
		this.questionType = value;
	}
	public get Options(): Option[] {
		return this.options;
	}
	public set Options(value: Option[]) {
		this.options = value;
	}
}